src/workloads/execution/CreateBigIndex.yml

```diff
@@ -90,4 +90,7 @@ Actors:
 
 AutoRun:
   Requires:
-    mongodb_setup: [single-replica]
+    mongodb_setup:
+    - standalone
```

Comment:
THis is a separate ticket and already pushed. I'm not sure why it's showing up here. Same with next ticket. I think it was just stupid on my local setup.